apps/studio/src/features/users/atom.ts

```diff
@@ -0,0 +1,30 @@
+import { RoleType } from "~prisma/generated/generatedEnums"
```

Comment:
nit: rename file to "atoms.ts" to be consistent

Reply:
updated in https://github.com/opengovsg/isomer/pull/1142/commits/f180c6cdd52b05faacfcd48f330bcf1eb39ca6a7